examples/ph_example/custom_model/custom.py

```diff
@@ -0,0 +1,68 @@
+import torch
+from torch.nn import functional as F
+from torchvision import datasets, transforms
+import os
+import PIL
+
+
+class CustomModel(torch.nn.Module):
```

Comment:
This might be a nitpick but can we rename these to be something specific to MNIST? something like "MnistCustomModel"? Because this is really just an example showing how you can create a custom model and **not** an actual Custom Model class you can use,.